server/lib/validation/languages/tok.js

```diff
@@ -0,0 +1,70 @@
+const tokenizeWords = require('talisman/tokenizers/words');
+
+const TRANSLATION_KEY_PREFIX = 'TRANSLATION_KEY:';
```

Comment:
Once adjusted (see below), this variable can be removed.

Reply:
Thanks. Done.